mxnet/training/docker/1.8.0/py3/cu110/Dockerfile.gpu

```diff
@@ -209,6 +210,12 @@ ADD https://raw.githubusercontent.com/aws/deep-learning-containers/master/src/de
 
 RUN chmod +x /usr/local/bin/deep_learning_container.py
 
+RUN HOME_DIR=/root \
+ && curl -o ${HOME_DIR}/oss_compliance.zip https://asimov-utility-packages.s3.amazonaws.com/oss_compliance.zip \
+ && unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ \
+ && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
+ && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR}
```

Comment:
Are we intentionally not deleting the downloaded zip file and the folder to which the zip file is extracted, or can we at least delete the zip file?